src/modules/blockchain/implementation/web3-service.js

```diff
@@ -871,6 +871,21 @@ class Web3Service {
         return Number(assertionChunksNumber);
     }
 
+    async getAssertionData(assertionId) {
+        const assertionData = await this.callContractFunction(
+            this.AssertionStorageContract,
+            'getAssertion',
+            [assertionId],
+        );
+        return {
+            ...assertionData,
```

Comment:
Why would we spread the result here?

Reply:
return looks like this : {
'0': BigNumber { _hex: 0x659ff179', _isBigNumber: true },
'1': BigNumber { _hex:
'0x012b', _isBigNumber: true },
'2': 4,
'3': BigNumber { _hex:
'0x04', _isBigNumber: true },
timestamp: 1704980857,
size: 299,
triplesNumber: 4,
chunksNumber: 4
}

Reply:
We only need

timestamp: 1704980857,
size: 299,
triplesNumber: 4,
chunksNumber: 4

No?

Reply:
Yes i agree. Even timestamp is excess in my opinion as it is not used in ValidationService